specifyweb/frontend/js_src/lib/components/DataModel/businessRuleDefs.ts

```diff
@@ -596,4 +596,16 @@ export const businessRuleDefs: MappedBusinessRuleDefs = {
       },
     },
   },
+  Preparation: {
+    onRemoved: (preparation, collection): void => {
+      if (preparation.get('isOnLoan') === true) {
+        setSaveBlockers(
+          collection.related ?? preparation,
+          preparation.specifyTable.field.isOnLoan,
+          [resourcesText.deleteLoanedPrep()],
+          resourcesText.loanedPrepDeletionMessage()
```

Comment:
The 4th param of `setSaveBlockers` needs to be a string key defined in `businessRuleUtils.ts` rather than a localization string
https://github.com/specify/specify7/blob/e257da0d53cf886f8d3cf6e2d3de3085ac13c1a6/specifyweb/frontend/js_src/lib/components/DataModel/businessRuleUtils.ts#L5-L10

Nothing will change functionally but would help keep things consistent!